tests/e2e/specs/wordpress/embedBlock.js

```diff
@@ -100,7 +100,10 @@ describe('Embed Block', () => {
 
       await setPostContent(EMBED_BLOCK_CONTENT);
 
+      // Increase timeout to stop publishing post failing this test.
+      await page.setDefaultTimeout(10000);
```

Comment:
Should this logic be moved to `publishPost`?

Reply:
Yeah, I thought about that. Make our own wrapper around this function, happy to make this change.